pwndbg/gdblib/disasm/arch.py

```diff
@@ -521,6 +523,39 @@ def _telescope_format_list(self, addresses: List[int], limit: int, emu: Emulator
                 enhance_string_len=enhance_string_len,
             )
 
+    @staticmethod
+    def _syscall_name(number: int, arch: str) -> str | None:
```

Comment:
I think this change is wrong; seems like not correct rebase/conflict merge/handling? :/

Reply:
Which change in particular? This function is needed by `_enhance_syscall` and to my knowledge, pwntools does not provide a function to convert syscall numbers into names.